dotCMS/src/main/java/com/dotcms/security/secret/ServiceSecrets.java

```diff
@@ -0,0 +1,80 @@
+package com.dotcms.security.secret;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ServiceSecrets implements Serializable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)